jest-common/src/main/java/io/searchbox/cluster/TasksInformation.java

```diff
@@ -0,0 +1,42 @@
+package io.searchbox.cluster;
+
+import io.searchbox.action.GenericResultAbstractAction;
+
+/**
+ * Currently only supports retrieving information for a particular task
+ */
+public class TasksInformation extends GenericResultAbstractAction {
```

Comment:
The result of a TaskInformation action is org.elasticsearch.tasks.TaskResult, which does not extend JestResult as required by AbstractAction.